### PR TITLE
update-k8s-reference-f34d425045363fa9b3741baa163afc5960aec67a

### PR DESCRIPTION
**Pull request by apparitor**
update-k8s-reference-f34d425045363fa9b3741baa163afc5960aec67a

### CODE REVIEW

File: content/docs/deploying/k8s/reference.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ PomeriumSpec defines Pomerium-specific configuration parameters.
 
 ### `authenticate`
 
-Authenticate sets authenticate service parameters
+Authenticate sets authenticate service parameters. If not specified, a Pomerium-hosted authenticate service would be used.
```
Comment (cmo-pomerium) [inline content/docs/deploying/k8s/reference.md:188]:
```suggestion
Authenticate sets authenticate service parameters. If not specified, a Pomerium-hosted authenticate service will be used.
```

**Review by cmo-pomerium — APPROVED** [review]
Quick change

File: content/docs/deploying/k8s/reference.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ PomeriumSpec defines Pomerium-specific configuration parameters.
                 
                 </p>
                 <p>
-                    <strong>Required.</strong>&#160;
-                    Authenticate sets authenticate service parameters
+                    
+                    Authenticate sets authenticate service parameters. If not specified, a Pomerium-hosted authenticate service would be used.
```
[the remark this answers is not in the document]
Comment (wasaga) [reply content/docs/deploying/k8s/reference.md:37]: those changes need be reflected upstream in the ingress-controller repo. I'll close this PR and fix it there.

**Comment by wasaga** [comment]
those changes need be reflected upstream in the ingress-controller repo. I'll close this PR and fix it there.

**Pull request closed without merging**